roles/wisdom_warmup_model/templates/wisdom-warmup_entrypoint.yml.j2

```
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wisdom-warmup-entrypoint
+  namespace: {{ wisdom_namespace }}
+data:
+  entrypoint.sh: |-
+    #! /bin/bash
```

you should add this here for safety:
```suggestion
    #! /bin/bash
    set -o errexit
    set -o pipefail
    set -o nounset
    set -o errtrace
    set -x
```